### PR TITLE
New Monocular Depth Estimation docs guide

### PR DESCRIPTION
**Pull request by Bovey0809**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Adds a comprehensive Monocular Depth Estimation (MDE) documentation page for YOLO11-MDE, including training/validation/prediction guides and supporting images. 📚🖼️

### 📊 Key Changes
- New docs page: `docs/en/tasks/mde.md` introducing YOLO11-MDE task, dataset format, metrics, and usage (Python/CLI)
- Added illustrative assets: prediction samples and training curves under `docs/en/tasks/images/mde/`
- Included model table with YOLO11n/s/m/l/x-mde performance and KITTI references

### 🎯 Purpose & Impact
- Guides users to train, validate, and predict with YOLO11-MDE on datasets like KITTI
- Clarifies MDE annotation format and depth normalization requirements
- Improves discoverability of depth + detection workflows, enabling real-time applications in robotics and ADAS
- Enhances docs quality with visual examples and concrete results ✅

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @username, thank you for submitting an `ultralytics/ultralytics` 🚀 PR! This is an automated review to help streamline the process—an Ultralytics engineer will assist shortly. To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA/) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_  — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing/). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

File: docs/en/tasks/mde.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+        metrics.box.map  # map50-95 for object detection
+        metrics.box.map50  # map50
+        metrics.box.map75  # map75
+        metrics.box.maps  # a list contains map50-95 of each category
+
+        # Access depth estimation metrics
+        print(f"Depth Error Rate: {metrics.depth_error:.2f}%")
+        print(f"Depth MAE: {metrics.depth_mae:.2f}m")
+        print(f"Depth RMSE: {metrics.depth_rmse:.2f}m")
+        print(f"Depth Accuracy (δ<1.25): {metrics.depth_acc:.2f}%")
```
Comment (UltralyticsAssistant) [inline docs/en/tasks/mde.md]: ⚠️ **HIGH**: The example suggests `model.val()` returns attributes like `metrics.depth_error`, `metrics.depth_mae`, `metrics.depth_rmse`, and `metrics.depth_acc`, but these properties are not exposed on the validation metrics object. Following this snippet will raise attribute errors for readers. Please update the example to reflect the actual structure/naming used for depth metrics (for example, accessing the `metrics.depth` namespace or the keys returned by `metrics.results_dict`).

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Please fix the validation example so it reflects the real depth metrics API; the current attribute names will raise errors for users running the snippet.

💬 Posted 1 inline comment

📋 **Skipped 5 files** (lock files, minified, images, etc.)

**Comment by glenn-jocher** [comment]
@Bovey0809 this looks good! 

We want to move the pictures somewhere else though, only code/text belongs in the repos. You can place them in the Docs assets like the other images here:
https://github.com/ultralytics/docs/releases/tag/0
Screenshot 2025-10-23 at 5 46 25 PM

**Comment by Bovey0809** [comment]
> @Bovey0809 this looks good!
> 
> We want to move the pictures somewhere else though, only code/text belongs in the repos. You can place them in the Docs assets like the other images here: https://github.com/ultralytics/docs/releases/tag/0 Screenshot 2025-10-23 at 5 46 25 PM

Sure, now I moved the images to the release v0.0.0 repo, and update the docs, thanks for telling me that trick.

**Comment by glenn-jocher** [comment]
@Bovey0809 looks better! We just need a couple other updates before merging:

- Let's use the full "Monocular Depth Estimation" in the filename which turns into a slug, so `/monocular-depth-estimation.md`
- We need to add new docs pages to mkdocs.yml and to the /index.md page above them so it shows up in the guides list in https://docs.ultralytics.com/guides

File: docs/en/tasks/monocular-depth-estimation.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+[Models](https://github.com/ultralytics/ultralytics/tree/main/ultralytics/cfg/models) download automatically from the latest Ultralytics [release](https://github.com/ultralytics/assets/releases) on first use.
+
+| Model       | size<br><sup>(pixels) | mAP<sup>val</sup><br>50-95 | params<br><sup>(M) | FLOPs<br><sup>(B) |
+| ----------- | --------------------- | -------------------------- | ------------------ | ----------------- |
+| YOLO11n-mde | 640                   | 51.9                       | 2.6                | 6.6               |
+| YOLO11s-mde | 640                   | 62.6                       | 9.5                | 21.7              |
+| YOLO11m-mde | 640                   | 64.9                       | 20.1               | 68.5              |
+| YOLO11l-mde | 640                   | 53.1                       | 25.4               | 87.6              |
+| YOLO11x-mde | 640                   | 63.6                       | 57.0               | 196.0             |
```
Comment (UltralyticsAssistant) [inline docs/en/tasks/monocular-depth-estimation.md]: 💡 **MEDIUM**: The table header opens multiple `<sup>` tags (`size<br><sup>(pixels)`, `params<br><sup>(M)`, `FLOPs<br><sup>(B)`) without closing them, so the remainder of the table renders in superscript. Please close each `<sup>` with `</sup>` to keep the table formatting readable.

File: docs/en/tasks/monocular-depth-estimation.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+        # Build and train a new YOLO11n-mde model from scratch
+        model = YOLO("yolo11-mde.yaml")  # automatically uses 'n' scale
+        results = model.train(data="kitti_mde.yaml", epochs=100, imgsz=640)
+        ```
+
+    === "CLI"
+
+        ```bash
+        # Build a new model from YAML and start training from scratch
+        yolo mde train data=kitti_mde.yaml model=yolo11-mde.yaml epochs=100 imgsz=640
```
Comment (UltralyticsAssistant) [inline docs/en/tasks/monocular-depth-estimation.md:52]: ⚠️ **HIGH**: `yolo11-mde.yaml` is not a shipped config, so these Python and CLI snippets will raise a “file not found” error. Please reference an actual model config such as `yolo11n-mde.yaml` (and mention swapping to other scales) here and in the later FAQ examples so users can run the commands successfully.

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review 2

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Document looks great overall, but please fix the broken `<sup>` tags in the model table and update the training snippets to point at real YOLO11-MDE configs like `yolo11n-mde.yaml`—right now the examples would fail because `yolo11-mde.yaml` doesn’t exist.

💬 Posted 2 inline comments
